misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -167,12 +166,16 @@ class JettyService @Inject internal constructor(
     server.stopTimeout = 25_000
     ServerConnectionStatistics.addToAllConnectors(server)
 
+    gzipHandler.server = server
     if (webConfig.gzip) {
-      gzipHandler.server = server
       gzipHandler.minGzipSize = webConfig.minGzipSize
       gzipHandler.addIncludedMethods("POST")
-      servletContextHandler.gzipHandler = gzipHandler
+    } else {
+      // GET is enabled by default for gzipHandler.
+      gzipHandler.addExcludedMethods("GET")
     }
+    gzipHandler.inflateBufferSize = 8192
```

Comment:
I guess this was the missing piece to allow requests?

Reply:
yep.

Reply:
yep.